EqualityAsserts.cs

```diff
@@ -170,7 +201,29 @@ public static void StrictEqual<T>(T expected, T actual)
 		{
 			Equal(expected, actual, EqualityComparer<T>.Default);
 		}
-
+#if XUNIT_ASSERT_ARRAY_AS_SPAN
+		/// <summary>
+		/// Verifies that two arrays of unmanaged type T are not equal, using Span&lt;T&gt;.SequenceEqual.
+		/// </summary>
+		/// <typeparam name="T">The type of items whose arrays are to be compared</typeparam>
+		/// <param name="expected">The expected value</param>
+		/// <param name="actual">The value to be compared against</param>
+		/// <exception cref="NotEqualException">Thrown when the arrays are equal</exception>
+#if XUNIT_NULLABLE
+		public static void NotEqual<T>([AllowNull] T[] expected, [AllowNull] T[] actual)
+#else
+		public static void NotEqual<T>(T[] expected, T[] actual)
+#endif
+			where T : unmanaged, IEquatable<T>
+		{
+			if (expected == null && actual == null)
+				throw new NotEqualException(ArgumentFormatter.Format(expected), ArgumentFormatter.Format(actual));
+			if (expected == null || actual == null)
+				return;
+			if (expected.AsSpan().SequenceEqual(actual))
+				throw new NotEqualException(ArgumentFormatter.Format(expected), ArgumentFormatter.Format(actual));
```

Comment:
This needs to be changed to Assert.NotEqual, like was done for Assert.Equal above.